gcp/plugin.go

```diff
@@ -20,6 +20,9 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 	p := &plugin.Plugin{
 		Name:             pluginName,
 		DefaultTransform: transform.FromCamel(),
+		DefaultGetConfig: &plugin.GetConfig{
+			ShouldIgnoreError: isIgnorableError([]string{"404", "400"}),
```

Comment:
What does `404` and `400` error code stand for?

Reply:
`404` is the not found error code and `400` is server error.